projects/bp-gallery/src/components/views/show-more/ShowMoreView.tsx

```diff
@@ -20,9 +22,11 @@ const ShowMoreView = ({
   categoryType: string;
   categoryId?: string;
 }) => {
-  const { linkToCollection, moveToCollection, removeFromCollection, bulkEdit } =
+  const { linkToCollection, moveToCollection, removeFromCollection, bulkEdit, addToExhibition } =
```

Comment:
this is currently unused, did you forget to add it to the bulkOperations of the PictureScrollGrid here?